Validation/trigger.fcl

```diff
@@ -42,3 +42,33 @@ physics.trigger_paths : [@sequence::Trig_valMenu.trigger_paths]
 physics.producers.EWMProducer.SpillType : 1
 #include "Production/JobConfig/common/epilog.fcl"
 #include "Production/JobConfig/reco/drop_trigger.fcl"
+
+
+physics.producers.aprHelixTriggerInfoMerger.ntupMode                : 1
```

Comment:
Didnt the name of that parameter change? Is a change in this PR to follow that needed?

Reply:
thanks for spotting this. I simply forgot the last `git push`. Now everything should be fine. I tested it locally successfully.